Sources/Widgets/Widgets3D/LineWidget/behavior.js

```diff
@@ -334,12 +335,12 @@ export default function widgetBehavior(publicAPI, model) {
     }
 
     if (
-      model.isDragging === false &&
+      isDragging === false &&
```

Comment:
do you know why an exact test on false when `isDragging` is initialized to `null`. This may be something to provide comment for

Reply:
The line in the condition (`publicAPI.rotateHandlesToFaceCamera();`) should be called each time the camera moves.
Even if this currently works (because we set `model.isDragging` to false after placing the first handle), I think it would be safer to use `!model.isDragging` instead of an exact test.